gilmour.go

```diff
@@ -25,6 +26,11 @@ func Get(backend Backend) *Gilmour {
 	return &x
 }
 
+type RetryConf struct {
```

Comment:
Why is this not a part of Gilmour itself?
RetryTimeout and RetryFrequency?

Btw. Shouldn't it be called RetryInterval instead of RetryFrequency?